examples/dmd/dg_advection.cpp

```diff
@@ -571,11 +571,11 @@ int main(int argc, char *argv[])
             sout << "parallel " << num_procs << " " << myid << "\n";
             sout.precision(precision);
             sout << "solution\n" << *pmesh << *u;
-            sout << "pause\n";
+            //sout << "pause\n";
```

Comment:
Should we just delete the commented lines rather than commenting them out?

Reply:
I vote for removing the pauses. What do you think? @dylan-copeland @chldkdtn

Reply:
I am okay with deleting.